jupyterhub/handlers/base.py

```diff
@@ -247,7 +249,14 @@ def clear():
             return
         cookie_id = cookie_id.decode('utf8', 'replace')
         self.log.info("[BaseHandler] Retrieving user with cookie id %s", cookie_id)
-        u = self.db.query(orm.User).filter(orm.User.cookie_id==cookie_id).first()
+
+        # Workaround for https://jira.corp.adobe.com/browse/PLATML-875
+        try:
```

Comment:
I suspect there is a better, more general place we could do this.  However, it appears that if we find ourselves in this situation, every call goes through this so will get us the needed restart.